R/dplyr-verbs.R

```diff
@@ -199,11 +199,17 @@ summarise.tbl_ts <- function(.data, ...) {
 
 #' @importFrom dplyr group_by_drop_default
 #' @export
-group_by.tbl_ts <- function(.data, ..., add = FALSE,
-                            .drop = group_by_drop_default(.data)) {
+group_by.tbl_ts <- function(.data, ..., .add = FALSE,
+                            .drop = group_by_drop_default(.data),
+                            add = FALSE) {
+  if (!missing(add)) {
+    warning("The `add` argument of `group_by()` is deprecated as of dplyr 1.0.0. Please use `.add`.")
+    .add <- add
+  }
```

Comment:
Can you remove the warning? The `NextMethod()` in the later part of the function should trigger a warning from dplyr.

Reply:
Dependency and remote entry added, and removed the warning.